Completed term entry for JS .padStart() (#4739)
Codecademy/docs: content/javascript/concepts/strings/terms/padStart/padStart.md

> ---
> Title: '.padStart()'
> Description: 'Takes an original string and returns a second string to the front in order to change the original string to a specified length.'
> Subjects:
>   - 'Web Development'
>   - 'Computer Science'
> Tags:
>   - 'Strings'
>   - 'Methods'
> CatalogContent:
>   - 'introduction-to-javascript'
>   - 'paths/front-end-engineer-career-path'
> ---

Reviewer: ```suggestion
---
Title: '.padStart()'
Description: 'Returns a new string of a specified length with the original string padded from the start with a given string.'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Methods'
  - 'Strings'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---
```

> Takes an original string and returns a second string to the front in order to change the original string to a specified length.

Reviewer: ```suggestion
In JavaScript, the **`.padStart()`** method returns a new string of a specified length, with the original string padded at the beginning with a given string (repeated if necessary) until the desired length is reached.
```

## Syntax

```pseudo

> originalString.padStart(targetLength, padString);

Reviewer: ```suggestion
string.padStart(targetLength, padString);
```

```

> - `originalString` starting string value before padding.
> - `targetLength` indicates the length the resulting string should be.
> - `padString`, (optional) character or string value to pad the original string.

Reviewer: ```suggestion
- `string`: The original string to be padded.
- `targetLength`: The desired length of the resulting string after padding.
- `padString`: The string to pad the original string with. If not provided, it defaults to a space (' ').
```

## Examples

Pad a string with spaces:

```js
const string = `Hi There!`;

const paddedString = string.padStart(15);

console.log(paddedString);
// Output:       Hi There!
```

Pad a string with 0s:

```js
const numString = `4457`;

const paddedNumString = numString.padStart(9, 0);

console.log(paddedNumString);
// Output: 000004457
```

Pad a string with a string less than the target length:

```js
const string = 'help!';
const targetLength = 21;
const padString = 'Can you '

const paddedString = string.padStart(targetLength, padString);

console.log(paddedString);
// Output: Can you Can you help!
```

Pad a string with a string more than the target length:

```js
const string = 'help!';
const targetLength = 9
const padString = 'Can you '

const paddedString = string.padStart(targetLength, padString);

console.log(paddedString);
// Output: Can help!

> ```

Reviewer: I would suggest putting all these use cases in one example, you can have multiple `console.log` showcasing multiple outputs.
After this bloc, use the ```shell``` block to show the output, avoid writing the output comments in the code itself. You can have comments in the code to explain the particular padding.

## Codebyte Example

> The following is runnable, and demonstrates the use of the `.padStart()` method:

Reviewer: ```suggestion
The following example demonstrates the use of the `.padStart()` method:
```
